pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -704,12 +704,7 @@ func configureVifV6Addresses(p *MasqueradePodInterface, err error) error {
 }
 
 func (p *MasqueradePodInterface) startDHCP(vmi *v1.VirtualMachineInstance) error {
-	addr := fmt.Sprintf("%s/24", p.vif.Gateway.String())
-	fakeServerAddr, err := netlink.ParseAddr(addr)
-	if err != nil {
-		return fmt.Errorf("failed to parse address while starting DHCP server: %s", addr)
-	}
-	return Handler.StartDHCP(p.vif, fakeServerAddr, p.bridgeInterfaceName, p.iface.DHCPOptions)
+	return Handler.StartDHCP(p.vif, p.vif.Gateway, p.bridgeInterfaceName, p.iface.DHCPOptions)
```

Comment:
I guess the only difference that I see relates to the address being validated before used.

Is this validation occurs in previous steps?

Reply:
[yes](https://github.com/kubevirt/kubevirt/blob/3052547b9261eee101ca013c86e21c02684cdbe6/pkg/virt-launcher/virtwrap/network/podinterface.go#L653).

Reply:
Thanks.